schema/divisions/division_area.yaml

```diff
@@ -26,6 +26,15 @@ properties:     # JSON Schema: Top-level object properties.
     allOf:
       - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
       - "$ref": ../defs.yaml#/$defs/propertyContainers/namesContainer
+    anyOf:
+      - properties:
+          is_land:
+            const: true
+        required: [is_land]
+      - properties:
+          is_territorial:
+            const: true
+        required: [is_territorial]
     properties: # JSON Schema: properties within GeoJSON top-level object 'properties' property
       subtype: { "$ref": "./defs.yaml#/$defs/propertyDefinitions/placetype" }
       class:
```

Comment:
`class` should be removed in favor of this new properties?

Reply:
same with division_boundary type

Reply:
Same comment here: We chatted about this - we're planning to keep division_area and division_boundary `class` values as is for now and only add `class` for division features. Once area and boundary features are updated, we'll update these.